tests/test_genome.py

```diff
@@ -1,11 +1,14 @@
-import time
-import urllib
 import json
 import numpy as np
 import pandas as pd
+import os
 
 from guidescanpy.flask.core.genome import get_genome_structure
 
+path_to_test_data = os.path.join(
+    os.path.dirname(os.path.abspath(__file__)), "genome_data"
```

Comment:
you may not need the `os.path.abspath` here

Reply:
You can put all the data files in the `data` folder, and use the argument `data_folder` in the test functions. (`data_folder` is a fixture available to all tests).